integration/get-set-delete.test.ts

```diff
@@ -157,4 +158,83 @@ describe('get/set/delete', () => {
     const getMiss = await Momento.get(IntegrationTestCacheName, cacheKey);
     expect(getMiss).toBeInstanceOf(CacheGet.Miss);
   });
+
+  it('should return INVALID_ARGUMENT_ERROR for invalid ttl when set with string key/value', async () => {
+    const setResponse = await Momento.set(
+      IntegrationTestCacheName,
+      v4(),
+      v4(),
+      {ttl: -1}
+    );
+    expect(setResponse).toBeInstanceOf(CacheSet.Error);
+    if (setResponse instanceof CacheSet.Error) {
+      expect(setResponse.errorCode()).toEqual(
+        MomentoErrorCode.INVALID_ARGUMENT_ERROR
+      );
+    }
+  });
+
+  it('should return INVALID_ARGUMENT_ERROR for invalid ttl when set with byte key/value', async () => {
+    const setResponse = await Momento.set(
+      IntegrationTestCacheName,
+      new TextEncoder().encode(v4()),
+      new TextEncoder().encode(v4()),
+      {ttl: -1}
+    );
+    expect(setResponse).toBeInstanceOf(CacheSet.Error);
+    if (setResponse instanceof CacheSet.Error) {
+      expect(setResponse.errorCode()).toEqual(
+        MomentoErrorCode.INVALID_ARGUMENT_ERROR
+      );
+    }
+  });
```

Comment:
Key/value conversion is separate from TTL validation. We shouldn't need to test them in combination.

Reply:
I had them only with string key/value to test TTL but when I checked our .NET SDK, we tested all the combinations (string key/value, byte key/value, string key/byte value, etc).
But I don't mind focusing on a specific test criteria here 👍 (which is TTL for this case)